src/main/java/uk/ac/ebi/atlas/experimentpage/json/opentargets/EvidenceService.java

```diff
@@ -401,13 +390,13 @@ private JsonObject associationRecord(JsonObject uniqueAssociationFields,
         return result;
     }
 
-    private Map<Contrast, DiseaseAssociation> getDiseaseAssociations(DifferentialExperiment experiment) {
-        Map<Contrast, DiseaseAssociation> result = new HashMap<>();
-        for (Contrast contrast : experiment.getDataColumnDescriptors()) {
+    private ImmutableMap<Contrast, DiseaseAssociation> getDiseaseAssociations(DifferentialExperiment experiment) {
+        var result = ImmutableMap.<Contrast, DiseaseAssociation>builder();
```

Comment:
In some places, I found a contextual name for the result. but in a few places, I found generic words as a 'result.
line 394,382..etc. It would be nice if we add a contextual name for the result here as well.

Reply:
I fixed all these generic names wherever I could find them. Hope that makes things more readable.